app/helpers/spree/admin/base_helper_decorator.rb

```diff
@@ -21,6 +21,14 @@ def link_to_remove_fields(name, f, options = {})
 
         link_to_with_icon('icon-trash', name, '#', html_options) + f.hidden_field(:_destroy)
       end
+
+      def add_check_if_single(mode, count)
+        if mode == :new && count == 1
+          { checked: true }
+        else
+          {}
+        end
+      end
```

Comment:
this is not a good location for this method.
this should probably go into a new helper, I see it's not easy to find a good name for it as it is shared between shipping and payment methods, I'd maybe go for helpers/admin/enterprises_helper.rb

Reply:
I have put the helper method in a new helper

Reply:
👍